src/styled-thing/index.js

```diff
@@ -97,6 +97,12 @@ export default function styled(Component) {
   }
 
   function StyledComponentFactory(styles) {
+    if (Array.isArray(styles)) {
```

Comment:
Can we make this only in DEV?

Reply:
Only as "performance improvements" since it will throw some error anyway

Reply:
yes, that's what I mean. Less code to run on prod = better

Reply:
Agreed